sagafalabella/sagafalabella/spiders/falabella.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from urllib.parse import quote_plus
+import json
+from scrapy import Request
+
+class FalabellaSpider(scrapy.Spider):
+    name = 'falabella'
+    allowed_domains = ['falabella.com.pe/falabella-pe']
+    start_urls = ['http://falabella.com.pe/falabella-pe']
+
+    def parse(self, response):
+        # we are not interested in some anchors give us url to all categories
+        categories = response.xpath(
+            '//ul[@class="fb-masthead__grandchild-links"]/li/a[not(contains(text(),"Ver todo"))]/@href').extract()
+        for categorie in categories:
```

Comment:
You should change `categorie` to `category`, as `categorie` doesn't exist :see_no_evil:, you could accordingly in the file?